tests/conftest.py

```diff
@@ -18,14 +19,22 @@ def load_array_from_zip_file(zip_file, file_name):
 def spike_test_pair():
     return load_array_from_zip_file("data/spike_test_pair.txt.zip", "spike_test_pair.txt")
 
+
 @pytest.fixture()
 def spike_test_var_dt():
     return load_array_from_zip_file("data/spike_test_var_dt.txt.zip", "spike_test_var_dt.txt")
 
+
 @pytest.fixture()
 def spike_test_high_init_dvdt():
     return load_array_from_zip_file("data/spike_test_high_init_dvdt.txt.zip", "spike_test_high_init_dvdt.txt")
 
+
+@pytest.fixture(scope="session")
+def tmp_nwb_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp("test_nwb")
+
```

Comment:
By using a unique temporary directory for every session we will always redownload the files or?

Reply:
I believe it does. I am not really happy with this myself. Any suggestions for improvements?

Reply:
Maybe this might be helpful? https://github.com/omarkohl/pytest-datafiles

Didn't realize that your test files would be really big.

Reply:
How about just downloading the files into the `tests/data` folder?

Reply:
And we already do that for the commit files as in `os.path.join(os.path.dirname(__file__), 'data')`.

Reply:
@sgratiy Sounds good.

Reply:
If you go this route, you should add a check for these files in the relevant tests and have pytest raise an 'expected fail' if they aren't where they should be.

Reply:
@sgratiy I've uploaded the files which I added to `tests/data`.

Reply:
@t-b where have you uploaded them to? I cannot find them on dropbox.

Reply:
I have changed all nwb tests to use fixtures with parameterization per @njmei  suggestion (thanx). There are no more nwb files in the bin directory. 

I had to do a merge because of some conflicts.